apps/core/scenes/passengerForm/PassengerForm.js

```diff
@@ -113,7 +113,7 @@ export default class PassengerForm extends React.Component<Props, State> {
             placeholder="Select"
             confirmLabel="OK"
             label="Nationality"
-            labelContainerStyle={styles.inputLabel}
+            containerStyle={styles.inputLabel}
```

Comment:
One more renaming here please 🙏 😉

Reply:
Done! Renamed all the Pickers (ios, android, web) props.